### PR TITLE
Describe Data Studios template image versioning

### PR DESCRIPTION
**Pull request by jason-seqera**
- https://seqera.atlassian.net/browse/PLAT-829

### CODE REVIEW

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 1]
```diff
@@ -30,6 +30,18 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
 
 Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the session container is built. You can install additional packages as needed during a session.
 
+The image template names include the version of the included analysis application and the Seqera image template version itself. The Seqera version string is of the following format:
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:33]:
```suggestion
The image template names include the version of the analysis application and the Seqera image template version itself. The Seqera version string looks like this:
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 2]
```diff
@@ -30,6 +30,18 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
+
+```ignore title="Image template version"
+<tool_version>-<seqera_version>-[u-<update_version>]
+```
+
+The following version strings apply:
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:39]:
```suggestion
```

Comment (justinegeffen) [reply platform_versioned_docs/version-24.2/data_studios/index.mdx:39]: I don't know if this suggestion comes through but I would leave out the introductory sentence.

Comment (jason-seqera) [reply platform_versioned_docs/version-24.2/data_studios/index.mdx:39]: @justinegeffen, just a hanging list under the example?

**Comment by jason-seqera** [comment]
Heh, as usual this wasn't actually complete and ready for review, but review happen ;)

**Comment by robnewman** [comment]
> Heh, as usual this wasn't actually complete and ready for review, but review happen ;)

@jason-seqera: LMK via tagging me when you want this reviewed!

**Comment by jason-seqera** [comment]
@robnewman, this can be reviewed now. Because we're only displaying the major.minor Seqera version in the UI, is the extended missive about tagging useful at all? I don't know that anyone is ever going to pull the images directly? Maybe as a basis for a custom container?

Thanks!

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 3]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
-You can also create your own container image templates. For more information, see [Custom environments][custom-envs].
+The image template tag includes the version of the analysis application and the Seqera image template version itself. The tag string looks like this:
+
+```ignore title="Image template tag"
+<tool_version>-[u<update_version>]-<seqera_version>
+```
+
+- `<tool_version>`: Analysis application version, such as `4.2.5` for JupyterLab
+- `<update_version>`: Seqera image template update integer version, such as `u1`, for instances where a breaking change is introduced
+- `<seqera_version>`: Seqera template image version, such as `0.7.0`
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:41]:
```suggestion
- `<connect_version>`: Seqera Connect client version, such as `0.7` or `0.7.0`.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 4]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
 
 Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the session container is built. You can install additional packages as needed during a session.
 
-You can also create your own container image templates. For more information, see [Custom environments][custom-envs].
+The image template tag includes the version of the analysis application and the Seqera image template version itself. The tag string looks like this:
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:33]:
```suggestion
The image template tag includes the version of the analysis application, an optional incompatibility flag, and the Seqera Connect version. Connect is the proprietary Seqera webserver client that manages communication with the container. The tag string looks like this:
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 5]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
+
+```ignore title="Image template tag"
+<tool_version>-[u<update_version>]-<seqera_version>
+```
+
+- `<tool_version>`: Analysis application version, such as `4.2.5` for JupyterLab
+- `<update_version>`: Seqera image template update integer version, such as `u1`, for instances where a breaking change is introduced
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:40]:
```suggestion
- `<update_version>`: Optional analysis application update version, such as `u1`, for instances where a backwards incompatible change is introduced.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 6]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
 Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the session container is built. You can install additional packages as needed during a session.
 
-You can also create your own container image templates. For more information, see [Custom environments][custom-envs].
+The image template tag includes the version of the analysis application and the Seqera image template version itself. The tag string looks like this:
+
+```ignore title="Image template tag"
+<tool_version>-[u<update_version>]-<seqera_version>
+```
+
+- `<tool_version>`: Analysis application version, such as `4.2.5` for JupyterLab
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:39]:
```suggestion
- `<tool_version>`: Third party analysis application that follows it's own semantic versioning `<major>.<minor>.<patch>`, such as `4.2.5` for JupyterLab.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 7]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
+- `<update_version>`: Seqera image template update integer version, such as `u1`, for instances where a breaking change is introduced
+- `<seqera_version>`: Seqera template image version, such as `0.7.0`
+
+Additionally, the Seqera version string looks like this:
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:43]:
```suggestion
Additionally, the Seqera Connect client version string has the format:
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 8]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
+
+```ignore title="Seqera version tag subset"
+<major>.<minor>.<patch>
+```
+
+- `<major>`: Signifies breaking changes in the underlying Seqera container
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:49]:
```suggestion
- `<major>`: Signifies major version changes in the underlying Seqera Connect client.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 9]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
+- `<minor>`: Signifies non-breaking changes in the underlying Seqera container
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:50]:
```suggestion
- `<minor>`: Signifies breaking changes in the underlying Seqera Connect client.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 10]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
+- `<patch>`: Signifies minor non-breaking changes in the underlying Seqera container
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:51]:
```suggestion
- `<patch>`: Signifies patch (non-breaking) changes in the underlying Seqera Connect client.
```

Comment (robnewman) [reply platform_versioned_docs/version-24.2/data_studios/index.mdx:51]: It's a little confusing to call the `<minor>` version "non-breaking" and `<patch>` version "minor". Also, the minor version is actually a breaking change.

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 11]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
+
+When pushed to the container registry, an image template is tagged with the following tags:
+
+- `<major>.<minor>`, such as `4.2.3-0.7`. When adding a new data studio this tag is the tag displayed in Seqera Platform.
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:55]:
```suggestion
- `<tool_version>-<major>.<minor>`, such as `4.2.3-0.7`. When adding a new data studio container template image this is the tag displayed in Seqera Platform.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 12]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
+- `<major>.<minor>.<patch>`, such as `4.2.3-0.7.1`.
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:56]:
```suggestion
- `<tool_version>-<major>.<minor>.<patch>`, such as `4.2.3-0.7.1`.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 13]
```diff
@@ -58,20 +83,28 @@ The default user is the `root` account. The following [conda-forge](https://cond
 
 To install additional Python packages, execute `!pip install <packagename>` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:84]:
```suggestion
To install additional Python packages during a running session, execute `!pip install <packagename>` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 14]
```diff
@@ -58,20 +83,28 @@ The default user is the `root` account. The following [conda-forge](https://cond
...
 
+To see the list of all JupyterLab image templates available, see [public.cr.seqera.io/repo/platform/data-studio-jupyter](https://public.cr.seqera.io/repo/platform/data-studio-jupyter).
+
 **RStudio Server 4.4.1**
 
 The default user is the `root` account. To install R packages, execute `install.packages("<packagename>")` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:90]:
```suggestion
The default user is the `root` account. To install R packages during a running session, execute `install.packages("<packagename>")` commands in your notebook environment. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 15]
```diff
@@ -58,20 +83,28 @@ The default user is the `root` account. The following [conda-forge](https://cond
...
 
+To see the list of all RStudio Server image templates available, see [public.cr.seqera.io/repo/platform/data-studio-rstudio](https://public.cr.seqera.io/repo/platform/data-studio-rstudio).
+
 **Visual Studio Code 1.93.1**
 
 [Visual Studio Code][vsc] is an integrated development environment (IDE) that supports many programming languages. The default user is the `root` account. To install extensions, select **Extensions**. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:96]:
```suggestion
[Visual Studio Code][vsc] is an integrated development environment (IDE) that supports many programming languages. The default user is the `root` account. To install extensions during a running session, select **Extensions**. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 16]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
+- `<major>`: Signifies breaking changes in the underlying Seqera container
+- `<minor>`: Signifies non-breaking changes in the underlying Seqera container
+- `<patch>`: Signifies minor non-breaking changes in the underlying Seqera container
+
+When pushed to the container registry, an image template is tagged with the following tags:
+
+- `<major>.<minor>`, such as `4.2.3-0.7`. When adding a new data studio this tag is the tag displayed in Seqera Platform.
+- `<major>.<minor>.<patch>`, such as `4.2.3-0.7.1`.
+
+To see the latest versions of the images, see [public.cr.seqera.io](https://public.cr.seqera.io/). You can also create your own container image templates. For more information, see [Custom environments][custom-envs].
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:58]:
```suggestion
To view the latest versions of the images, see [public.cr.seqera.io](https://public.cr.seqera.io/). You can also augment the Seqera-provided image templates or use your own custom container image templates. This approach is recommended for managing reproducible analysis environments. For more information, see [Custom environments][custom-envs].
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 17]
```diff
@@ -58,20 +83,28 @@ The default user is the `root` account. The following [conda-forge](https://cond
...
 
 [Visual Studio Code][vsc] is an integrated development environment (IDE) that supports many programming languages. The default user is the `root` account. To install extensions, select **Extensions**. Additional system-level packages can be installed in a terminal window using `apt install <packagename>`.
 
+To see the list of all Visual Studio Code image templates available, see [public.cr.seqera.io/platform/data-studio-vscode](https://public.cr.seqera.io/repo/platform/data-studio-vscode).
+
 **Xpra 6.2.0**
 
 [Xpra](https://github.com/Xpra-org/xpra), known as _screen for X_, allows you to run X11 programs by giving you remote access to individual graphical applications. The container template image also installs NVIDIA Linux x64 (AMD64/EM64T) drivers for Ubuntu 22.04 for running GPU-enabled applications. To use these GPU drivers, your compute environment must specify GPU instance families.
 
 The default user is the `root` account. The image is based on `ubuntu:jammy`. Additional system-level packages can be installed in a terminal window using `apt install <package_name>`.
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:104]:
```suggestion
The default user is the `root` account. The image is based on `ubuntu:jammy`. Additional system-level packages can be installed during a running session in a terminal window using `apt install <package_name>`.
```

**Review by robnewman — CHANGES_REQUESTED** [review]
Looks good - just some minor changes. Thanks!

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 18]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
...
 Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the session container is built. You can install additional packages as needed during a session.
 
-You can also create your own container image templates. For more information, see [Custom environments][custom-envs].
+The image template tag includes the version of the analysis application, an optional incompatibility flag, and the Seqera Connect version. Connect is the proprietary Seqera webserver client that manages communication with the container. The tag string looks like this:
+
+```ignore title="Image template tag"
+<tool_version>-[u<update_version>]-<seqera_version>
+```
+
+- `<tool_version>`: Third party analysis application that follows it's own semantic versioning `<major>.<minor>.<patch>`, such as `4.2.5` for JupyterLab.
```
Comment (justinegeffen) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:39]:
```suggestion
- `<tool_version>`: Third-party analysis application that follows its own semantic versioning `<major>.<minor>.<patch>`, such as `4.2.5` for JupyterLab.
```

File: platform_versioned_docs/version-24.2/data_studios/index.mdx [hunk 19]
```diff
@@ -30,7 +30,32 @@ Currently, Data Studios only supports AWS Batch compute environments that **do n
 
 Data Studios provides four container image templates: JupyterLab, RStudio Server, Visual Studio Code, and Xpra. The image templates install a very limited number of packages when the session container is built. You can install additional packages as needed during a session.
 
-You can also create your own container image templates. For more information, see [Custom environments][custom-envs].
+The image template tag includes the version of the analysis application, an optional incompatibility flag, and the Seqera Connect version. Connect is the proprietary Seqera webserver client that manages communication with the container. The tag string looks like this:
+
+```ignore title="Image template tag"
+<tool_version>-[u<update_version>]-<seqera_version>
```
Comment (robnewman) [inline platform_versioned_docs/version-24.2/data_studios/index.mdx:36]:
```suggestion
<tool_version>-[u<update_version>]-<connect_version>
```

**Review by robnewman — APPROVED** [review]
Just one minor change please!

**Pull request merged**